### PR TITLE
Add project and package

### PR DESCRIPTION
**Pull request by batesyuk**

### CODE REVIEW

File: govuk-mobile-ios-onboarding/govuk-mobile-ios-onboarding/AppDelegate.swift [hunk 1]
```diff
@@ -0,0 +1,36 @@
+//
+//  AppDelegate.swift
+//  govuk-mobile-ios-onboarding
+//
+//  Created by Thomas Bates on 18/04/2024.
+//
```
Comment (gclssvglx) [inline govuk-mobile-ios-onboarding/govuk-mobile-ios-onboarding/AppDelegate.swift]: Seems somewhat redundant given the git history and context.

File: govuk-mobile-ios-onboarding/govuk-mobile-ios-onboarding/SupportingFiles/SceneDelegate.swift [hunk 2]
```diff
@@ -0,0 +1,52 @@
+//
+//  SceneDelegate.swift
+//  govuk-mobile-ios-onboarding
+//
+//  Created by Thomas Bates on 18/04/2024.
+//
```
Comment (gclssvglx) [inline govuk-mobile-ios-onboarding/govuk-mobile-ios-onboarding/SupportingFiles/SceneDelegate.swift]: Same as before...

File: govuk-mobile-ios-onboarding/govuk-mobile-ios-onboarding/ViewControllers/ViewController.swift [hunk 3]
```diff
@@ -0,0 +1,20 @@
+//
+//  ViewController.swift
+//  govuk-mobile-ios-onboarding
+//
+//  Created by Thomas Bates on 18/04/2024.
+//
```
Comment (gclssvglx) [inline govuk-mobile-ios-onboarding/govuk-mobile-ios-onboarding/ViewControllers/ViewController.swift]: Oh, there's lots of these 🙀

File: Example/Production/govuk_onboarding_ios/SupportingFiles/SceneDelegate.swift [hunk 4]
```diff
@@ -1,8 +1,8 @@
 //
-//  SceneDelegate.swift
-//  govuk-mobile-ios-onboarding
+// GOV.UK
 //
-//  Created by Thomas Bates on 18/04/2024.
+// Copyright © 2024 Government Digital Services.
+// All Rights Reserved.
 //
```
Comment (gclssvglx) [inline Example/Production/govuk_onboarding_ios/SupportingFiles/SceneDelegate.swift]: As [mentioned](https://github.com/alphagov/govuk-mobile-ios-services/pull/3#discussion_r1579135241) - not something we'd normally do.

File: GovUK-Onboarding.xcworkspace/xcshareddata/IDETemplateMacros.plist [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>FILEHEADER</key>
+	<string>
+// GOV.UK
+//
+// Copyright © ___YEAR___ ___ORGANIZATIONNAME___.
+// All Rights Reserved.
+//</string>
```
Comment (gclssvglx) [inline GovUK-Onboarding.xcworkspace/xcshareddata/IDETemplateMacros.plist:11]: Might be something to take advice on - if you haven't already. We usually let the MIT license take care of this I think.

File: Example/GovUK-Onboarding.xcodeproj/project.pbxproj [hunk 6]
```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 56;
+	objectVersion = 60;
```
Comment (gclssvglx) [inline Example/GovUK-Onboarding.xcodeproj/project.pbxproj]: Should this be the same as [here](https://github.com/alphagov/govuk-mobile-ios-services/pull/3#discussion_r1579141376)?

File: Onboarding/Sources/Onboarding/Onboarding.swift [hunk 7]
```diff
@@ -0,0 +1,2 @@
+// The Swift Programming Language
+// https://docs.swift.org/swift-book
```
Comment (gclssvglx) [inline Onboarding/Sources/Onboarding/Onboarding.swift:2]: Same as [this](https://github.com/alphagov/govuk-mobile-ios-services/pull/3#discussion_r1579137605)

File: Onboarding/Tests/OnboardingTests/OnboardingTests.swift [hunk 8]
```diff
@@ -0,0 +1,12 @@
+import XCTest
+@testable import Onboarding
+
+final class OnboardingTests: XCTestCase {
+    func testExample() throws {
+        // XCTest Documentation
+        // https://developer.apple.com/documentation/xctest
+
+        // Defining Test Cases and Test Methods
+        // https://developer.apple.com/documentation/xctest/defining_test_cases_and_test_methods
```
Comment (gclssvglx) [inline Onboarding/Tests/OnboardingTests/OnboardingTests.swift:10]: Same as [this](https://github.com/alphagov/govuk-mobile-ios-services/pull/3#discussion_r1579139027)

File: Package.swift [hunk 9]
```diff
@@ -21,8 +21,8 @@ let package = Package(
         ),
         .testTarget(
             name: "OnboardingTests",
-            path: "Onboarding/Tests",
-            dependencies: ["Onboarding"]
+            dependencies: ["Onboarding"],
+            path: "Onboarding/Tests"
```
Comment (gclssvglx) [inline Package.swift]: Order matters here? Seems odd for named variables (kinda defeats the purpose somewhat I think)

File: Fastlane/Fastfile [hunk 10]
```diff
@@ -4,7 +4,6 @@ require_relative 'configuration'
 
 default_platform :ios
 
-# rubocop:disable Metrics/BlockLength
```
Comment (gclssvglx) [inline Fastlane/Fastfile]: Surprised this no longer complains 😁

File: .github/workflows/pull_request.yaml [hunk 11]
```diff
@@ -26,6 +26,9 @@ jobs:
         with: 
           bundler-cache: true
 
+      - name: Select xcodes
+        run: xcodes select
+
```
Comment (gclssvglx) [inline .github/workflows/pull_request.yaml]: Same as [this](https://github.com/alphagov/govuk-mobile-ios-services/pull/3#discussion_r1579144399)

File: Example/GovUK-Onboarding.xcodeproj/project.pbxproj [hunk 12]
```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 60;
+	objectVersion = 56;
```
Comment (gclssvglx) [inline Example/GovUK-Onboarding.xcodeproj/project.pbxproj:6]: Same as [this](https://github.com/alphagov/govuk-mobile-ios-services/pull/3#discussion_r1579141376)

File: GovUK-Onboarding.xcworkspace/xcshareddata/IDETemplateMacros.plist [hunk 13]
```diff
@@ -12,4 +12,4 @@
 	<key>ORGANIZATIONNAME</key>
 	<string>Government Digital Services</string>
```
Comment (gclssvglx) [inline GovUK-Onboarding.xcworkspace/xcshareddata/IDETemplateMacros.plist:13]: Same as [this](https://github.com/alphagov/govuk-mobile-ios-services/pull/3#discussion_r1579135241)

File: .github/workflows/pull_request.yaml [hunk 14]
```diff
@@ -26,7 +26,7 @@ jobs:
         with: 
           bundler-cache: true
 
-      - name: Select xcodes
+      - name: Select xcode
         run: xcodes select
```
Comment (gclssvglx) [inline .github/workflows/pull_request.yaml]: 😀  https://github.com/alphagov/govuk-mobile-ios-services/pull/3#discussion_r1579144399

File: .github/workflows/pull_request.yml
Comment (gclssvglx) [inline .github/workflows/pull_request.yml:1]: Still love it 💚 https://github.com/alphagov/govuk-mobile-ios-services/pull/3#discussion_r1579148386

**Review by gclssvglx — APPROVED** [review]
Happy if you want to merge this @batesyuk 👍

Do have some questions which show my ignorance of all things iOS and Swift, but no blockers.

File: govuk-mobile-ios-onboarding/govuk-mobile-ios-onboarding/AppDelegate.swift (hunk 1 above)
Comment (batesyuk) [reply govuk-mobile-ios-onboarding/govuk-mobile-ios-onboarding/AppDelegate.swift]: Not sure what has happened here, this code is out of date. Same as the other repo, I will remove headers going forward

**Pull request merged**